contracts/test/forkMainnet/AMMWrapper/CollectFee.t.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "contracts-test/forkMainnet/AMMWrapper/Setup.t.sol";
+import "contracts-test/utils/BalanceSnapshot.sol";
+
+contract TestAMMWrapperCollectFee is TestAMMWrapper {
+    using BalanceSnapshot for BalanceSnapshot.Snapshot;
+
+    // Received same amount as expected min amount
```

Comment:
These comments are almost identical to the test case name, maybe could be removed